tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -130,9 +128,25 @@ host: '<host>', port: 4000,user: '<username>', password: '<your_password>', data
 </div>
 </SimpleTab>
 
-## Where is the CA root path on my system?
+## Root certificate management
+
+### Root certificate issuance and validity
+
+TiDB Serverless Tier uses certificates from [Let's Encrypt](https://letsencrypt.org/) as a Certificate Authority (CA) for TLS connection between clients and TiDB Serverless Tier clusters. Usually, the root certificate ([ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt)) of Let's Encrypt is present in your system's root CA stores. The Serverless Tier Server digital certificates issued by Let's Encrypt are valid for 90 days and will be automatically rotated upon expiration.
+
+> **Note:**
+>
+> TiDB Serverless Tier does not provide a CA root certificate download, because we don't guarantee that the same CA will be used to issue a certificate in the future, which will cause the CA root certificate to change.
+
+If the client uses the system's root CA stores by default, such as Java and Go, you can easily connect securely to TiDB Serverless Tier clusters without specifying the path of CA roots.If you need the CA certificate of a TiDB Serverless Tier cluster,Please download the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) instead of the single CA certificate.
```

Comment:
```suggestion
If the client uses the system's root CA stores by default, such as Java and Go, you can easily connect securely to TiDB Serverless Tier clusters without specifying the path of CA roots. If you still want to get a CA certificate for a TiDB Serverless Tier cluster, you can download and use the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) instead of a single CA certificate.
```